tests/integration/test_data/cached_dependencies.yaml

```diff
@@ -887,3 +887,84 @@ yarn_cached_deps:
     - "pkg:npm/is-positive@3.1.0"
     - "pkg:npm/pathval@1.1.1"
     - "pkg:npm/type-detect@4.0.8"
+rubygems_cached_deps:
+  # Use local version of repos.
+  # <use_local: True> is not supported in local environment and will be skipped.
+  use_local: True
```

Comment:
@ejegrova Thanks for moving those testing repos. I started working on integration test for cached dependencies, but I'm a little confused. Those tests aren't executed neither by the pipeline nor the local dev environment, because of these lines of code (I think), so I don't know how to make sure those tests work and what's the point of having tests that aren't executed?

Reply:
I believe that these tests make commits to repos under the cachito-testing organization and require a credential in order to do so. Therefore they only get executed in our internal test environment and not upstream/locally.